### PR TITLE
DGC-176: Creating the hero styles for the style guide.

### PR DESCRIPTION
**Pull request by krapp91**
https://drupal4gov.atlassian.net/browse/DGC-176

### CODE REVIEW

File: docroot/themes/custom/twentyeighteen/sass/components/hero/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+.hero-block {
+  width: 100%;
+}
+
+.hero-background-img {
+  position: relative;
+}
+
+.hero-logo {
+  display: none;
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/hero/_hero.scss]: Why are we doing display: none for smaller widths?

File: docroot/themes/custom/twentyeighteen/sass/components/hero/hero.twig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class= "hero-block">
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/hero/hero.twig]: Can we re-do this markup to be more BEM style?

File: docroot/themes/custom/twentyeighteen/sass/components/hero/_hero.scss (hunk 1 above)
Comment (krapp91) [reply docroot/themes/custom/twentyeighteen/sass/components/hero/_hero.scss]: @lkopacz from the mocks in https://drupal4gov.atlassian.net/browse/DGC-3 there is a logo image floating in the middle of the background image on larger screen sizes. On the small width mock, the logo is gone so I hid it with display: none. Is there a better way to do this?

File: docroot/themes/custom/twentyeighteen/sass/components/hero/hero.twig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class= "hero">
+  <div class= "hero__background-img">
+    <img src="../images/wide_home-hero.jpg"/>
+  </div>
+  <div class= "hero__logo">
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/hero/hero.twig]: The only thing I would change here markup wise is have the logo and the button under one wrapper. Maybe hero__details, hero__detatils-logo, hero__details-callout...mostly so with absolute positioning you don't have to redo several elements for different breakpoints.

**Comment by lkopacz** [comment]
@krapp91 This does look good, but for the sake of finishing out a deadline, I am closing this in place of https://github.com/Drupal4Gov/Drupal-GovCon-2017/pull/584. I needed to make a few extra tweaks to this and couldn't push up to your remote.

**Pull request closed without merging**